.github/workflows/ci.yaml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on:
+  # Run this on pushes to main
+  push:
+    branches:
+    - main
+
+  # Or when PR operations are done
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+
+jobs:
+  validate-test-helm-v3:
+    name: Validate Helm tests with Helm V3 on a KinD Cluster
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        kube-tag:
+          - v1.18.2
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.3.0
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.2.1
```

Comment:
I think we can skip install of Helm 3.2.1, since 3.3.0 is also installed?

Reply:
yes, we don't need to downgrade.